packages/phone-number-privacy/common/src/interfaces/responses.ts

```diff
@@ -35,19 +37,16 @@ export interface SignMessageResponseSuccess extends PnpQuotaStatus {
   version: string
   signature: string
   warnings?: string[]
-  // TODO(2.0.0) audit req/res types - https://github.com/celo-org/celo-monorepo/issues/9804
-  // - warnings not provided in 1.1.11
 }
 
 export interface SignMessageResponseFailure {
   success: false
   version: string
   error: string
+  // These fields are occasionally provided by the signer but not the combiner
```

Comment:
Hmm do you mean why we haven't nested the field (i.e. difficultues restructuring the request/response types) or why the combiner doesn't return these values?

If it's the latter, it has to do with how the combiner separates & processes success/failure responses anyways. If the signer response fails, then it's irrelevant if that signer returned quota values since these won't be used in the quota calculation anyways. Changing this would be a much more involved change

Reply:
Ahh yeah for sure!